client/state/data-layer/wpcom/read/recommendations/sites/index.js

```diff
@@ -0,0 +1,67 @@
+
+/**
+ * External Dependencies
+ */
+import { map } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import {
+	READER_RECOMMENDED_SITES_REQUEST
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { receiveRecommendedSites } from 'state/reader/recommended-sites/actions';
+
+export function requestRecommendedSites( { dispatch }, action, next ) {
+	const { seed = 1, number = 10, offset = 0 } = action.payload;
+	dispatch( http( {
+		method: 'GET',
+		path: '/read/recommendations/sites',
+		query: { number, offset, seed, posts_per_site: 0 },
```

Comment:
@blowery, @samouri: adding the WP_Site object should be really easy. What attribute in the [WP_Site object](https://developer.wordpress.org/reference/classes/wp_site/) do we need here?

Sample WP_Site object:

> WP_Site Object
> (
>     [blog_id] => 6705221
>     [domain] => figjamandlimecordial.wordpress.com
>     [path] => /
>     [site_id] => 1
>     [registered] => 2009-02-23 22:39:23
>     [last_updated] => 2017-04-24 08:49:47
>     [public] => 1
>     [archived] => 0
>     [mature] => 0
>     [spam] => 0
>     [deleted] => 0
>     [lang_id] => 1
>     [ds_blog] => 33
>     [ds_stats] => 0
>     [blogname] => Fig Jam and Lime Cordial
>     [siteurl] => http://figjamandlimecordial.wordpress.com
>     [post_count] => 1345
>     [home] => http://figjamandlimecordial.com
> )